go/plugins/googleai/googleai.go

```diff
@@ -40,6 +40,8 @@ type Config struct {
 	// Embedding models to provide.
 	// If empty, a complete list will be obtained from the service.
 	Embedders []string
+
+	ClientOptions []option.ClientOption
```

Comment:
This needs docs.  In particular that this replaces the `APIKey` field.

Reply:
Done.